FrontEnd/src/main/java/resourceDisplay/AssetsRepository.java

```diff
@@ -10,6 +10,31 @@
  */
 public class AssetsRepository {
 
+    /**
+     * This is used to create a list of all workspace values for a specific user
+     *
+     * @param dbClient    - A client for connection to the DB
+     * @param userID - The user for whom we want the list of workspaces
+     * @return a list of all workspace values for the specific user
+     */
+    public List<String> getWorkspaceIdList(DatabaseClient dbClient, String userID) {
+        List<String> workspaceIdList = new ArrayList<>();
+        String statementString = "SELECT DISTINCT workspaceId FROM Workspace_User_Table WHERE " +
+                "userEmail = @userID ORDER BY workspaceId";
+        Statement statement = Statement.newBuilder(statementString)
+                .bind("userID")
+                .to(userID)
+                .build();
+        try (ResultSet resultSet = dbClient.singleUse().executeQuery(statement)) {
```

Comment:
just noticed this - can't comment on lines that aren't added in this PR for some reason, but anyway, this comment is for line 3: importing using * has two disadvantages:
1. It obfuscates where various classes in the code originate, so it's harder to infer their type when reading.
2. It can create clashes (for example if you imported * from another package that had a ResultSet class).

It's better to import each class separately.

Reply:
Fixed this (also in IndexController).